doc/nrf/config_and_build/bootloaders_and_dfu/fw_update.rst

```diff
@@ -259,51 +259,73 @@ Recompile with ``priv_c.pem`` and the incremented firmware version to correctly
+  For example, if you opted for :ref:`ug_bootloader_adding_upgradable_mcuboot`, the application image versioning configuration is described in :ref:`ug_fw_update_image_versions_mcuboot`.
 
 .. _ug_fw_update_image_versions_mcuboot:
 
-Using MCUboot
-=============
+Configuring image version with MCUboot
+======================================
+
+To assign a semantic version number to your application when you have opted for booting application directly by the MCUboot bootloader (that is, if you have opted for either :ref:`ug_bootloader_adding_immutable_mcuboot` or :ref:`ug_bootloader_adding_upgradable_mcuboot`), add the version string to the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` option for the application:
+
+.. code-block:: console
 
-MCUboot supports two different implementations of downgrade protection:
+   CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION="0.1.2+3"
 
-* Software-based prevention, where the current version can be encoded into the currently active image.
-* Hardware-based prevention, where the current version can be encoded into a non-volatile storage partition outside of any image.
+See the `Semantic versioning`_ webpage or :doc:`mcuboot:imgtool` for details on version syntax.
 
 .. _ug_fw_update_image_versions_mcuboot_downgrade:
+.. _ug_fw_update_downgrade_protection:
 
-Software-based downgrade prevention
------------------------------------
+Downgrade protection
+********************
 
-The |NCS| supports MCUboot's software-based downgrade prevention for application images, using semantic versioning.
-This feature offers protection against any outdated firmware that is uploaded to a device.
+The downgrade protection feature blocks downgrading the firmware version on the device.
+This can be useful for security reasons, because firmware downgrade can allow attackers to bring back security vulnerabilities that were fixed by a particular firmware update.
 
-To enable this feature, set the configuration options ``CONFIG_MCUBOOT_DOWNGRADE_PREVENTION`` and ``CONFIG_BOOT_UPGRADE_ONLY`` for the MCUboot image.
+As part of this process, the bootloader compares the new firmware image version with the version of the currently booted firmware.
+The bootloader rejects the update if the new image version is lower than the version of the currently booted firmware.
 
-To assign a semantic version number to your application, pass the version string into the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` option for the application:
+The implementation is related to the :ref:`ug_fw_update_image_versions` and also depends on the type of bootloader used:
 
-.. code-block:: console
+* Software-based downgrade protection is supported only by MCUboot.
+  In this kind of protection, the current firmware version is encoded into the currently active image.
+* Hardware-based downgrade protection is supported by both MCUboot and |NSIB|.
```

Comment:
As far as I know it's not supported when doing multi-image 4 slots with network core in MCUBoot I don't know if we need to mention this here.

Reply:
We could I think, I suggest to handle it in a follow-up task. What you think @greg-fer ?